src/index.ts

```
@@ -42,68 +44,101 @@ export class PatchyInternetQImpl {
 	private queue: Queue<Action>;
 	private dlQueue: Queue<Action>;
 	private isListening = false;
-	private queueStatus : string = __IDLE__;
-	private readonly hooksRegistry;
-	private readonly transformerRegistry;
-	private persistence;
+	private queueStatus = __IDLE__;
+	private readonly hooksRegistry: Record<string, (payload: Object) => Promise<void>>;
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)